### PR TITLE
Radio buttons

### PR DESCRIPTION
**Pull request by ramshorst**
radio

### CODE REVIEW

**Comment by ramshorst** [comment]
Related to https://github.com/upfluence/ember-influencer/pull/14

File: less/base/_inputs.less [hunk 1]
```diff
@@ -146,3 +145,29 @@ label {
...
 }
+
+//
+// Radio buttons
+//
+.ufp-input--radio {
+  display: table;
+  width: 100%;
+
+    .btn.btn-primary {
```
Comment (AlexisMontagne) [inline less/base/_inputs.less]: why this ident?

Comment (ramshorst) [reply less/base/_inputs.less]: Good point ;)

**Comment by AlexisMontagne** [comment]
LGTM

File: less/base/_inputs.less [hunk 2]
```diff
@@ -146,3 +145,33 @@ label {
   -moz-appearance: none;
   appearance: none;
 }
+
+//
+// Radio buttons
+//
+.ufp-input--radio {
```
Comment (phndiaye) [inline less/base/_inputs.less]: Typo: `upf`

Comment (ramshorst) [reply less/base/_inputs.less]: thks!

File: less/base/_inputs.less [hunk 3]
```diff
@@ -146,3 +145,33 @@ label {
...
+  .btn.btn-primary {
+    border: 1px solid @upf-gray;
+    color: @upf-primary-rock-blue;
+    background-color: @upf-gray-light;
+    font-weight: 100;
+    font-family: @font-text-body;
+    box-shadow: none;
+    outline: none;
+    height: 37px;
+    line-height: 25px;
```
Comment (phndiaye) [inline less/base/_inputs.less]: Do we really need to change the line-height ? isn't it just like the buttons ?

File: less/base/_inputs.less [hunk 4]
```diff
@@ -146,3 +145,33 @@ label {
...
+    border-radius: @default-radius;
+
+      input{
```
Comment (phndiaye) [inline less/base/_inputs.less]: unwanted indentation ?

Comment (ramshorst) [reply less/base/_inputs.less]: thks!

File: less/base/_inputs.less (hunk 3 above)
Comment (ramshorst) [reply less/base/_inputs.less]: kind of yes :S

**Pull request merged**